### DISCUSSION TITLE
Team QBs

### DISCUSSION THREAD

**Discussion by bohsask** [110yards/fantasy#208] [Ideas]
Overall the site has been very stable and working great this season, so first of all I just wanted to congratulate you on a job well done on the site. 

One of the things our group has been discussing is that next season we would like to be able to use 'team qbs', i.e. instead of starting an individual player as a qb we would prefer to start a team's 'qb unit' and therefore receive the points for the unit as a whole as opposed to the individual player.   This was how the old fantaseh site managed the QBs and I think for a small league like the cfl with limited options (i.e. in our 8-team league all quarterbacks are used weekly), this is a more fair and balanced approach.  

When many team's QB positions are unsettled (as they are this year) and they are shifting between quarterbacks between games and within games, those without a top tier QB are at a big disadvantage.  And then you have teams with the strategy of substituting a backup on short yardage (i.e. vulturing TDs) which does not help either.

Likely the simplest way to accomplish this would be to have multiple QB spots allowed on the starting roster and have this configurable by the GM as all other starting spots are.  Currently Kickers, RBs and QBs are limited to 1.  Is there a technical reason for this?  Leagues using this would need a gentlemen's rule to prohibit rostering more than one 'starting' qb, or that the QBs rostered must be on the same team or something to that effect. 

Just wanted to get the discussion going on this and get it in the queue for potential enhancements for next year.

**Comment by mdryden** [comment]
I'm open to this idea, but I am concerned it might be tricky.

I'll explore it, but in the meantime, if you are interested in team QBs, please weigh in on this thread.
